lib/src/kratos_client.dart

```diff
@@ -636,24 +637,54 @@ class KratosClient {
     }
 
     try {
+      if (kIsWeb) {
```

Comment:
Let's refactor and wrap both code paths in methods, so that we end up with `if (kisWeb) await logoutWeb(); else logoutNative();` here. It will be much easier to understand.